stable/fluent-bit/templates/config.yaml

```diff
@@ -267,5 +267,7 @@ data:
 {{- end }}
 {{ end }}
 {{- end }}
-
+{{- if .Values.extraConfigMapEntries }}
+{{ .Values.extraConfigMapEntries | indent 2 }}
```

Comment:
Will this work when adding an object? Should't it be something like {{ toYaml .Values.extraConfigMapEntries | indent 2 }} ?

Reply:
It's a map, so you would use it like this:
```extraConfigMapEntries: |
        scripts.lua: |
          function cb_print(tag, timestamp, record)
            return code, timestamp, record
          end
```